pic-sure-hpds-ui/src/main/webapp/picsureui/psamaui/login/not_authorized.hbs

```diff
@@ -0,0 +1,7 @@
+<div id="not_authorized_message">
+    <h2>Not Authorized</h2>
+    <p>Your user account is not yet authorized to use this system.</p>
+    <p>Please contact the <a href="{{helpLink}}">administrator</a> if you believe this is an error.</p>
+    <p>If you believe you logged in using the wrong account, please <a
+            href="/psamaui/login/?redirection_url=/picsureui/">Click Here</a> to try logging in again.</p>
+</div>
```

Comment:
The logging out of users was formerly done right here. I moved that to right before this is rendered, in login.js